src/sentry/models/groupassignee.py

```diff
@@ -92,6 +116,22 @@ def deassign(self, group, acting_user=None):
             )
             activity.send_notification()
 
+        # sync Sentry assignee to external issues
+        if features.has('organizations:internal-catchall', group.organization, actor=acting_user):
```

Comment:
I think we should abstract this since it's repeated in both sections. Seems to be a common enough pattern for "sync all the external issues for a group."

Reply:
good call